consensus/polybft/validator/validator_config.go

```diff
@@ -0,0 +1,97 @@
+package validator
```

Comment:
Since it only contains `GenesisValidator` struct and its functions, I would rename this file to `genesis_validator.go`.